src/controllers/user.controller.js

```diff
@@ -278,6 +338,50 @@ class UserController {
     }
   }
 
+  /**
+   *
+   *
+   * @description This is method that resets the User password
+   * @param  {object} req The request object
+   * @param  {object} res The response object
+   * @returns {object} json response
+   */
+  static async reset(req, res) {
+    const { email } = req.currentUser;
+    try {
+      const reset = await BaseRepository.find(User, { email });
+      if (!reset) {
+        return responseGenerator.sendError(
+          res,
+          401,
+          `verification link not valid`
+        );
+      }
+      await BaseRepository.update(
+        User,
+        {
+          password: req.body.password
+        },
+        {
+          email
+        }
+      );
+      return responseGenerator.sendSuccess(
+        res,
+        200,
+        null,
+        'Password reset successful'
+      );
+    } catch (err) {
+      return responseGenerator.sendError(
+        res,
+        500,
+
```

Comment:
You can remove this additional empty line

Reply:
I think this comment is not ask and therefore not necessary owning to the fact that eslint cater for such